app/code/Magento/Wishlist/Model/Wishlist.php

```diff
@@ -486,17 +490,15 @@ public function addNewItem($product, $buyRequest = null, $forciblySetQty = false
      */
     public function setCustomerId($customerId)
     {
-        return $this->setData($this->_getResource()->getCustomerIdFieldName(), $customerId);
+        return $this->setData(self::CUSTOMER_ID, $customerId);
```

Comment:
This change may break 3d party extension that uses this extension point for customer_id field